src/fixup-svg-string.js

```diff
@@ -57,5 +57,8 @@ module.exports = function (svgString) {
     // Empty script tags and javascript executing
     svgString = svgString.replace(/<script[\s\S]*>[\s\S]*<\/script>/, '<script></script>');
 
+    // Remove hrefs that aren'ts data uris
+    svgString = svgString.replace(/href="(?!data).*"/, '');
```

Comment:
Might have to use a regex more like: `href\s*=\s*"(?!data)[^"]*"` to account for extra whitespace and to keep from greedily matching until the last quotes in the whole string